{"name": "test_getbyid_failure", "status": "passed", "start": 1728927427782, "stop": 1728927427783, "uuid": "6daaa53f-c128-441b-87db-d47d1451877b", "historyId": "cbc29a25fe64c1c9081b3685cd78d352", "testCaseId": "cbc29a25fe64c1c9081b3685cd78d352", "fullName": "Exercise.exercise_test.TestExerciseServiceWithoutMock#test_getbyid_failure", "labels": [{"name": "parentSuite", "value": "Exercise"}, {"name": "suite", "value": "exercise_test"}, {"name": "subSuite", "value": "TestExerciseServiceWithoutMock"}, {"name": "host", "value": "DESKTOP-IDGJ8R7"}, {"name": "thread", "value": "5496-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Exercise.exercise_test"}]}